lib/modules/manager/ocb/readme.md

```diff
@@ -0,0 +1,21 @@
+This manager allows updating dependencies defined in build definitions for the [OpenTelemetry Collector Builder (ocb)](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder).
+
+The `ocb` manager has no `fileMatch` default patterns, so it won't match any files until you configure it with a pattern.
+
+```json title="If your builder files are named like foo-builder.yml or builder.yaml"
```

Comment:
@HonkingGoose does this work with mkdocs?

Reply:
It should work. 🙂

See: https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-a-title